bitcoin/src/blockdata/script/mod.rs

```diff
@@ -142,18 +145,60 @@ pub fn write_scriptint(out: &mut [u8; 8], n: i64) -> usize {
     len
 }
 
+/// Returns minimally encoded scriptint as a byte vector.
+pub fn scriptint_vec(n: i64) -> Vec<u8> {
```

Comment:
In fd78f83e7e4f2fb6dcbd5ce8ca555b436973f72b:

This also feels like the wrong API. I'd almost rather just expose `write_scriptint`.

Reply:
Removed